libs/ledger-live-common/src/families/osmosis/js-signOperation.ts

```diff
@@ -99,7 +99,7 @@ const signOperation = ({
         const accountId = account.id;
 
         const fee = transaction.fees || new BigNumber(DEFAULT_FEES);
-        const extra = { memo: transaction.memo || {} };
+        const extra = { memo: transaction.memo || "" };
```

Comment:
Is this fixing an issue on LLD as well?

Reply:
Yeah I guess it was an edge case that only showed up in LLM. When memo is not defined we should pass an empty string instead of an empty object.

Reply:
So I guess your answer is "no impact on LLD" :)
If so this fine, otherwise we'd want to cherry-pick before the LLD release I think.

Reply:
Correct, this would have been caught during testing, but just to be sure I just tested it again specifically to be sure. No impact on LLD!